packages/files-ui/src/Components/Modules/FileBrowsers/views/FilesList.tsx

```diff
@@ -944,62 +1029,84 @@ const FilesList = ({ isShared = false }: Props) => {
                     <TableHeadCell>
-                      {/* Checkbox */}
-                    </TableHeadCell>
-                    <TableHeadCell
-                      align='left'
-                      onSortChange={toggleSortDirection}
-                      sortButtons
-                      sortDirection={direction}
-                    >
-                      {t`Name`}
-                    </TableHeadCell>
-                    <TableHeadCell align='right'>
-                      <Menu
-                        testId='fileDropdown'
-                        icon={<SortIcon className={classes.dropdownIcon} />}
-                        options={[{
-                          contents: (
-                            <ListItemText inset>
-                              <b><Trans>Sort By:</Trans></b>
-                            </ListItemText>
-                          )
-                        }, {
-                          contents: (
-                            <>
-                              {column === "name" && <ListItemIcon>
-                                <CheckIcon />
-                              </ListItemIcon>}
-                              <ListItemText inset={column !== "name"}>
-                                <Trans>Name</Trans>
-                              </ListItemText>
-                            </>
-                          ),
-                          onClick: () => setColumn("name")
-                        }, {
-                          contents: (
-                            <>
-                              {column === "date_uploaded" && <ListItemIcon><CheckIcon /></ListItemIcon>}
-                              <ListItemText inset={column !== "date_uploaded"}>
-                                <Trans>Date Uploaded</Trans>
-                              </ListItemText>
-                            </>
-                          ),
-                          onClick: () => setColumn("date_uploaded")
-                        }, {
-                          contents: (
-                            <>
-                              {column === "size" && <ListItemIcon><CheckIcon /></ListItemIcon>}
-                              <ListItemText inset={column !== "size"}>
-                                <Trans>Size</Trans>
-                              </ListItemText>
-                            </>
-                          ),
-                          onClick: () => setColumn("size")
-                        }]}
-                        style={{ focusVisible: classes.focusVisible }}
-                      />
+                      {selectedItems.length > 0 &&
+                        <CheckboxInput
+                          value={selectedItems.length === items.length}
+                          indeterminate={selectedItems.length > 0}
+                          onChange={toggleAll}
+                          testId="select-all"
+                        />
+                      }
```

Comment:
What do you think about showing this checkbox all the time to allow to select all also on mobile?
```suggestion
                      {<CheckboxInput
                        value={selectedItems.length === items.length}
                        indeterminate={selectedItems.length > 0}
                        onChange={toggleAll}
                        testId="select-all"
                      />
```